app.rb

```diff
@@ -0,0 +1,80 @@
+require './book'
+require './student'
+require './teacher'
+require './rental'
+
+def list_all_books(books)
+  if books.empty?
+    puts 'No book found!'
+  else
+    books.each { |book| puts "Title: '#{book.title}', Author: #{book.author}" }
+  end
+end
+
+def list_all_people(people)
+  if people.empty?
+    puts 'No person found!'
+  else
+    people.each { |person| puts "[#{person.class}] Name: #{person.name}, ID: #{person.id}, Age: #{person.age}" }
+  end
+end
+
+def create_person(*)
+  puts 'Do you want create student (1) or teacher (2)? [Enter the number]: '
+  person_type = gets.chomp.to_i
+  case person_type
+  when 1
+    print 'Enter student name: '
+    name = gets.chomp
+    print 'Enter age: '
+    age = gets.chomp
+    @people.push(Student.new('classroom', age, name))
+    puts 'Student created successfully'
```

Comment:
- [ ] While creating the age of a student there is an error `push` is not defined kindly fix it.
### Error
![sssss](https://user-images.githubusercontent.com/81727988/182881948-80708d86-0286-429d-8496-21be4f58fa63.png)